dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.expressionLanguage-4.0.feature

```diff
@@ -0,0 +1,7 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.jakarta.expressionLanguage-4.0
+singleton=true
+-bundles=io.openliberty.jakarta.expressionLanguage.4.0; location:="dev/api/spec/,lib/"; mavenCoordinates="org.apache.tomcat:tomcat-el-api:10.0.0-M7"
```

Comment:
Are those the correct maven co-ords? Isn't it `jakarta.el:jakarta.el-api:4.0.0` ?

Reply:
These are the correct coordinates, we pull the el api/impl from tomcat
 as they must come from the same source (some impl details in the api for EL).